src/s3a_the_gutter.tw2

```diff
@@ -367,10 +375,10 @@ Everyone at the bar was where I had left them. I got the feeling this bunch were
 
 He said nothing.
 
-(click: "nothing")[$dissolve[I waited. Still not a fucking peep.]]
+(click: "nothing")[$dissolve[I passed the time staring at a dusty, empty shelf behind Toe labeled "GAMES". Still not a fucking peep.]]
 
 (click: "peep")[(if: (text: $gameOfChance)'s length > 1)[$dissolve[I [[pulled out the $gameOfChance|The Gutter - Bar - Toe game start]]].]
-\(else:)[$dissolve[...I got [[bored|The Gutter - Bar hub]].]]]
+\(else:)[$dissolve[...I got [[bored.|The Gutter - Bar hub]]]]]
```

Comment:
Most online writing styles (and the passages I've worked on) have terminating punctuation outside of a hyperlink. That seems like something we should go through and make consistent throughout the game before we finalize it.

Reply:
Yeah, I've been doing that same thing normally and I'll add it to the punch list. In this case, the period was hanging before the text loaded in. So this was an attempt to fix that.